docs/source/markdown/podman-events.1.md

```diff
@@ -115,8 +115,10 @@ Format the output to JSON Lines or using the given Go template.
 | .Name                 | Container name (string)                       |
 | .Network              | Name of network being used (string)           |
 | .PodID                | ID of pod associated with container, if any   |
+| .Recycle PATH BOOL    | [unimplemented]                               |
```

Comment:
This is the new convention for documenting cobra functions. It merits careful scrutiny.